src/components/fields/ColorArrayPicker.js

```diff
@@ -0,0 +1,133 @@
+import Field from './Field';
```

Comment:
actually, I think this one is valid to put in a whole bunch of more places, basically anywhere where we have the MultiColorPicker, if should actually be this one, to allow to color by array, its an ability we didn't add before, and some clients have asked for it 

cc @nicolaskruchten : what do you think, should we make this generalizable and apply more widely?

Reply:
you're right that it could apply more widely but i hesitate a bit... it'll make the whole UI even heavier than it already is if we have all these constant-vs-variable toggles everywhere. can we create an issue and just do it here for now?

I think I'd like to have a smaller set of toggles for these conditional-widgets.